docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -35,7 +35,7 @@ be accessed from the operating system.
     in terminal:
 
 ```bash
-tpm2_pcrlist
+sudo tpm2_pcrread
```

Comment:
@IgorBagnucki We know for a fact that tpm2-tools change the names or options of the utilities quite often. I think you found out which was the version introducing this change? Maybe you could add a step with checking version of the tools. And then provide two opsions: tpm2_pcrlist for version > X, tpm2_pcrread for version <= X ?